src/ripple/app/misc/impl/Manifest.cpp

```diff
@@ -320,15 +310,7 @@ ManifestCache::load (
                 continue;
             }
 
-            if (unl.listed(mo->masterKey))
-            {
-                applyManifest (std::move(*mo), unl);
-            }
-            else
-            {
-                JLOG(j_.info())
-                   << "Manifest in db is no longer trusted";
-            }
+            applyTrustedManifest (std::move(*mo));
```

Comment:
Is it possible for the manifest to become untrusted after the prior shutdown?  (Not sure if that question is well-posed..)